src/elm/Page/Community/New.elm

```diff
@@ -491,10 +492,24 @@ update msg model loggedIn =
                             ]
                     }
 
-        GotCreateCommunityResponse (Ok ( symbol, subdomain )) ->
+        GotCreateCommunityResponse (Ok symbol) ->
             model
                 |> UR.init
-                |> UR.addExt (LoggedIn.CreatedCommunity symbol subdomain)
+                |> UR.addPort
+                    { responseAddress = msg
+                    , responseData = Encode.null
+                    , data =
+                        Eos.encodeTransaction
+                            [ { accountName = loggedIn.shared.contracts.community
+                              , name = "upsertrole"
+                              , authorization =
+                                    { actor = loggedIn.accountName
+                                    , permissionName = Eos.samplePermission
+                                    }
+                              , data = defaultRoleTransaction symbol
+                              }
+                            ]
```

Comment:
Oh, I thought that wouldn't work, but it does! 🤯

I thought we needed the community to be created first, and then create the role. But I'm not 100% sure on how it works:

1. It doesn't need the community to be created (does that mean there is no concept of foreign key on eos?)
2. It still creates the community first, because actions are run sequentially inside a transaction (if this is the right answer, is there a chance the `upsertrole` action will be executed first, and produce an error?)
3. Something else?

Please enlighten me 🙇

Reply:
it's number 2 bro! the nice thing about EOS is that it behaves the same as a regular database, regarding transactions. they will be executed in order and they will also rollback in case one of them fails.

Some decoders/encoders don't care about order in JSON, so this might be a problem in some languages, i think elm keeps the same order!

Reply:
Yeah, that's what I thought, that's nice! And yeah, I do believe Elm keeps it in the same order. At least in my testing, it worked fine